src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -60,16 +62,15 @@ export const SearchBarDropdownSection = ({
               isHovered={hoveredIndex === index + startingIndex}
               setHoveredIndex={setHoveredIndex}
               toggleOpen={toggleOpen}
+              index={index + startingIndex}
               traceEvent={() =>
```

Comment:
can you use the useTrace hook (example of how to use here:
https://github.com/Uniswap/interface/blob/449c323fbb50468f82b21422c369b3dc2ca267a0/src/components/Widget/index.tsx#L61) so we can preserve properties from the parent component's contexts? similar comment for the other use of sendAnalyticsEvent below

Reply:
yep, but you can include the useTrace properties with the other eventProperties, like this for example: 
` const trace = useTrace({ section: SectionName.WIDGET })
  const [initialQuoteDate, setInitialQuoteDate] = useState<Date>()
  const onInitialSwapQuote = useCallback(
    (trade: Trade<Currency, Currency, TradeType>) => {
      setInitialQuoteDate(new Date())
      const eventProperties = {
        // TODO(1416): Include undefined values.
        ...formatSwapQuoteReceivedEventProperties(
          trade,
          /* gasUseEstimateUSD= */ undefined,
          /* fetchingSwapQuoteStartTime= */ undefined
        ),
        ...trace,
      }
      sendAnalyticsEvent(EventName.SWAP_QUOTE_RECEIVED, eventProperties)
    },
    [trace]
  )`

from here: https://github.com/Uniswap/interface/blob/449c323fbb50468f82b21422c369b3dc2ca267a0/src/components/Widget/index.tsx#L61